data/html5video-rules.json

```diff
@@ -0,0 +1,32 @@
+{
+    "rules": [
+        {
+            "name": "MOV",
+            "displayName" : "MOV",
+            "icon": "",
+            "regex": "<a href="(https|http)?:?(.*\.mov)".*>.*<\/a>",
+            "replacement": "<div class='embed-container' data-src="$1" data-type="video/mp4" data-codec="avc1.42E01E, mp4a.40.2" ><br><video width="100%" height="auto" preload controls><source src="$1" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"' /></video></div>"
+        },
+        {
+            "name": "MP4",
+            "displayName" : "MP4",
+            "icon": "",
+            "regex": "<a href="(https|http)?:?(.*\.mp4)".*>.*<\/a>",
+            "replacement": "<div class="embed-container"  data-src="$1" data-type="video/mp4" data-codec="avc1.42E01E, mp4a.40.2" ><br><video width="100%" height="auto" preload controls><source src="$1" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"' /></video></div>"
```

Comment:
Did you try your solution with relative urls?

Example: `./videos/presentation.mp4`